src/components/box/box.style.ts

```diff
@@ -39,9 +39,14 @@ const StyledBox = styled.div.attrs(applyBaseTheme)<
   ${grid}
   ${calculatePosition}
 
-  ${({ borderRadius = "borderRadius000" }) => css`
-    border-radius: var(--${borderRadius});
-  `}
+${({ borderRadius = "borderRadius000" }) => {
+    const radiusValues = borderRadius.split(" ");
```

Comment:
suggestion (non-blocking): as we've done this a couple of times we could move it to a shared util